lib/puppet/util/servicenow.rb

```diff
@@ -392,6 +392,31 @@ def catalog_compilation_failure?(resource_statuses, transaction_completed)
     resource_statuses.empty? && !transaction_completed
   end
   module_function :catalog_compilation_failure?
+
+  def find_environment_match(environment, allow, block)
```

Comment:
Seeing as this returns a boolean I might add a question mark at the end of the function name. Like environment_match?